cmd/wstail/tail.go

```diff
@@ -288,7 +249,6 @@ func (r *processGroup) Close() (err error) {
 	for _, closer := range r.closers {
 		closer.Close()
 	}
-	r.terminate()
```

Comment:
What if it does not exit fast enough - e.g. if refreshing logs faster than usual - won't this leave (some) as a zombies?